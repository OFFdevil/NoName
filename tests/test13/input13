def FUNCa A A
{
    // body
}

main
  int A:=123;