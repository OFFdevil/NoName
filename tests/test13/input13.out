Function: name= FUNCa has already had parametr with this name!
